packages/data/walkthroughs/9.9.9.json

```diff
@@ -1555,8 +1555,230 @@
           "nextNavigateTo": "P11"
         }
       ]
-    }
-  },
+    }, 
+    "P18-9.9.9.3-remove-after-review": {
+      "walkthroughItemType": "multiChoice",
+      "questionText": "Is this dwelling unit located above another dwelling unit or common space?",
+      "possibleAnswers": [
+        {
+          "answerDisplayText": "Yes",
+          "answerValue": "true"
+        },
+        {
+          "answerDisplayText": "No",
+          "answerValue": "false"
+        }
+      ],
+      "nextNavigationLogic": [
+        {
+          "nextLogicType": "equals",
+          "answerToCheck": "P18",
+          "answerValue": "true",
+          "nextNavigateTo": "R1.1"
+        },
+        {
+          "nextLogicType": "fallback",
+          "nextNavigateTo": "P19"
+        }
+      ]
+    },
+    "P19": {
+      "walkthroughItemType": "multiChoice",
+      "questionText": "Does the egress door of this dwelling unit open onto a shared egress facility (corridor, passageway, stair, ramp, etc.) that is served by a single stair or ramp?",
+      "possibleAnswers": [
+        {
+          "answerDisplayText": "Yes",
+          "answerValue": "true"
+        },
+        {
+          "answerDisplayText": "No",
+          "answerValue": "false"
+        }
+      ],
+      "nextNavigationLogic": [
+        {
+          "nextLogicType": "and",
+          "valuesToCheck": [
+            {
+              "nextLogicType": "equals",
+              "answerToCheck": "P19",
+              "answerValue": "true"
+            },
+            {
+              "nextLogicType": "contains",
```

Comment:
Oooh, I don't see this "nextLogicType" used anywhere yet in @nicholaslyon-dt's current data transformation, but would probably be reasonable to be part of his implementation.

In my implementation [I literally checked every value](https://github.com/bcgov/House-Innovations-HOUSNAV/pull/15/files#r1621209253), but this is probably easier to manage. We can probably merge both, but just add it to the list of things that will need to be checked by Nick!

Reply:
I would vote to re-write the data to conform to the "or" logic type with a bunch of "equal" checks for each value. It keeps the javascript more simple (in that we're not adding an extra type to manage/test) but I'm open to adding this if we feel like we definitely need it this way.